### PR TITLE
1444 custom latlong not mandatory in patch

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Custom lat long no longer mandatory in patching

### CODE REVIEW

**Comment by hwindsor** [comment]
Code coverage < 50% as there are no lat long custom fields that are mandatory

File: app/modules/site_settings/helpers.py [hunk 1]
```diff
@@ -476,10 +476,14 @@ def is_valid_value(cls, cf_defn, value):
                 value[0] = float(value[0])
             if isinstance(value[1], int):
                 value[1] = float(value[1])
-            if not util.is_valid_latitude(value[0]):
+            if not value[0] and not value[1]:
```
Comment (naknomum) [inline app/modules/site_settings/helpers.py:479]: am i making this up, or does `not 0.0` evaluate to `True` in python?  if so, i think these may need to be made into `value[X] is not None`.   and i guess probably a good reason for us to add a test to confirm `[0, 0]` indeed passes.

**Review by naknomum — APPROVED** [review]
looks good

**Review by drewblount — APPROVED** [review]
LGTM

**Pull request merged**